externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/udf/KDFRegistry.scala

```diff
@@ -36,6 +37,14 @@ object KDFRegistry {
     "string",
     "1.3.0")
 
+  val engine_name: KyuubiDefinedFunction = create(
+    "engine_name",
+    udf(() => SparkContext.getOrCreate().getConf.get("spark.app.name")).asNonNullable(),
+    "Return the engine_name of Kyuubi Server",
```

Comment:
Return the  spark application name for the associated query engine

Reply:
Thansk for the suggestion.